packages/app/vite.config.mjs

```diff
@@ -1,4 +1,4 @@
-import { makeConfig } from '../frontend-shared/vite.config'
+import { makeConfig } from '../frontend-shared/vite.config.mjs'
```

Comment:
So Vite is all-in on ESM it seems, we cannot use `ts` like this. We either need `type: module`, or something... I'm not sure what, I think `vite.config.mts` should also work, except we need to update TypeScript, which is pretty involved. The least invasive change is to use `mjs` here.

Reply:
Good idea, done: https://github.com/cypress-io/cypress/issues/22945.

I think it's probably a config and/or TS version related issus.